app/Platform/RouteServiceProvider.php

```diff
@@ -74,6 +75,9 @@ protected function mapWebRoutes(): void
             Route::get('game/{game}/dev-interest', GameDevInterestController::class)->name('game.dev-interest');
             Route::get('game/{game}/suggest', [SuggestGameController::class, 'forGame'])->name('game.suggest');
 
+            Route::get('achievement/{achievement}/create-ticket', [AchievementController::class, 'createTicket'])->name('achievement.create-ticket');
```

Comment:
Given your question about the old tickets route, I've changed this to `achievement/{achievement}/tickets/create`